apps/pdf-viewer/webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const VueLoaderPlugin = require('vue-loader/lib/plugin')
+
+module.exports = {
+	plugins: [
+		new VueLoaderPlugin()
+	],
+	devtool: 'source-map',
+	entry: './src/app.js',
+	output: {
+		path: `${__dirname}/js`,
+		chunkFilename: '[name].pdf-viewer.chunk.js',
+		filename: "pdf-viewer.bundle.js",
+		publicPath: 'apps/pdf-viewer/js/'
```

Comment:
we should adjust this to fit the needs of the core webpack builder. publicPath should be 'apps/pdf-viewer/dist/' and path should be 'dist/'

Reply:
> we should adjust this to fit the needs of the core webpack builder. publicPath should be 'apps/pdf-viewer/dist/' and path should be 'dist/'

@scramb adjusted - please review again - THX